src/config/RichTextEditor/ToolbarButtons/Lighthouse.jsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import { injectLazyLibs } from '@plone/volto/helpers/Loadable/Loadable';
+import Icon from '@plone/volto/components/theme/Icon/Icon';
+import showSVG from '@plone/volto/icons/show.svg';
```

Comment:
Questa icona dell'occhio non mi convince perchè viene utilizzata in generale per mostrare e nascondere un contenuto, non vorrei che venisse fraintesa, suggerisco di utilizzare una di queste due icone
- un target, più generico
<img width="198" alt="Screenshot 2022-11-22 alle 12 19 53" src="https://user-images.githubusercontent.com/43245702/203302849-a05d90bf-3d7d-4a75-a110-f2b419caa60d.png">

- l'icona delle news che potrebbe vagamente assomigliare al faro di lighthouse però ditemi voi
<img width="198" alt="Screenshot 2022-11-22 alle 12 19 33" src="https://user-images.githubusercontent.com/43245702/203303457-8d1458bc-db28-4047-8685-a0fc373d26fe.png">